lib/actionizer/inputs.rb

```diff
@@ -23,17 +23,18 @@ def check_for_param_error(method_name, params = {})
           return "Param #{param} can't be nil"
         end
 
-        type = attrs.fetch(:type)
+        if attrs.fetch(:required) && !params.include?(param)
+          return "Param #{param} is required for #{method_name}"
+        end
 
-        # Type check if param was required, or if optional and passed in
-        if attrs.fetch(:required) || params.include?(param)
-          if type && !(params[param].class <= type)
-            return "Param #{param} must descend from #{type}"
-          end
+        if !params.include?(param)
+          next
```

Comment:
`next unless params.include?(param)` guard clause IMHO :) but not my repo, lol

Reply:
Unless is the worst

Reply:
@mikenichols specifically asked me to change it from `next unless` to a multi-line :(

Reply:
And it hurt me in my heart area.

Reply:
It's only because simplecov only does coverage by line, not by branch. I would prefer the shorter code too.